sources/php5/KalturaClientBase.php

```diff
@@ -706,7 +706,7 @@ public function setClientConfiguration(KalturaClientConfiguration $configuration
 		$params = get_class_vars('KalturaClientConfiguration');
 		foreach($params as $param)
 		{
-			if(is_null($configuration->$param))
+			if(empty($configuration->$param) || is_null($configuration->$param))
```

Comment:
See:
https://github.com/kaltura/clients-generator/pull/128/commits/993a338f5ff4d8fc9b1960e895de6a7e57fe7e67

Also, note that I changed the condition somewhat.. if I understand the intention here [and perhaps I do not:)], it is that if any of the KalturaClientConfiguration object members passed to setClientConfiguration() are null, they should be unset on $this->clientConfiguration[], however, since if is_null($var) returns true, isset($var) will necessarily return false, the unset() statement will never come to pass.